src/layouts/MoreMenu.tsx

```diff
@@ -56,7 +60,15 @@ export function MoreMenu() {
         keepMounted={true}
       >
         {moreNavigation.map((item, index) => (
-          <MenuItem component={Link} href={item.link} key={index}>
+          <MenuItem
+            component={Link}
+            href={
+              item.makeLink
+                ? item.makeLink({ cryptoSymbol: 'AAVE', network, walletAddress })
```

Comment:
Could you possibly remove the `cryptoSymbol` and `network` for this specific link?

On the dashboard and staking page it makes sense to pre-select the network and asset, but it would be nice to let users choose network/asset in the Transak modal when navigating from this button.

Reply:
Done